core/boxes_manager.rb

```diff
@@ -42,4 +42,23 @@ def getBox(key)
     @boxesManager[key]
   end
 
+  def getBoxByConfig(config_path, node_name)
```

Comment:
Do we need showBoxByConfig? I'm asking because I can not test it without some additional arguments like --template and --node-name. Threre need to be mdbci  command for that like ./mdbc box-by-template ... ?